orc8r/cloud/go/services/streamer/servicers/servicer.go

```diff
@@ -29,6 +30,10 @@ var (
 	stringifiedEAGAIN = fmt.Sprintf("%s", streamer.EAGAIN)
 )
 
+const (
+	defaultSubscriberBatchSize = 100
```

Comment:
Given that streamer is generic (i.e. support policies, apn, subscribers, etc.), let's remove the notion of a subscriber from this const.